### DISCUSSION TITLE
OpenAI Assistant generate graph request failed

### DISCUSSION THREAD

**Discussion by rcandelu** [FlowiseAI/Flowise#1201] [Q&A]
Hello, I received this error when ask to generete a graph: 

Error: ENOENT: no such file or directory, open '/root/.flowise/openai-assistant/443053d8-6e94-4269-9b6b-f8c338f1b600.png'

I'm using latest version of docker image (Flowise 1.4.0)
Could help me?

Thank you

**Comment by HenryHengZJ** [comment]
It was fixed in `1.4.2`, try reinstalling the latest

**Reply by rcandelu** [reply]
Confirmed thank you
